### PR TITLE
scripts for zcu102_revmin platform for scout

### PR DESCRIPTION
**Pull request by sivarajeshjarugula**
Scripts to generate the zcu102_revmin scout platform dsa and other custom files needed for platform

Signed-off-by: Siva Rajesh Jarugula <sivaraj@xilinx.com>

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

File: src/platform/zcu102_revmin/zcu102_revmin_pfm.tcl [hunk 1]
```diff
@@ -0,0 +1 @@
+zcu104_pfm.tcl
```
Comment (mamin506) [inline src/platform/zcu102_revmin/zcu102_revmin_pfm.tcl]: Please do not use link for this file. At lease the description of the platform is different.

Comment (sivarajeshjarugula) [reply src/platform/zcu102_revmin/zcu102_revmin_pfm.tcl]: Hi Min,
I will change it to be a file instead of softlink

**Comment by sivarajeshjarugula** [comment]
> 
> 
> Build Failed! :(

Can I understand what is meant by build failed?

**Comment by NaAl-Xlnx** [comment]
retest this please

**Comment by sivarajeshjarugula** [comment]
updated the tcl file to be zcu102_revmin_dsa.tcl and remove the softlink for pfm tcl file

**Pull request merged**